Sources/FrenchRepublicanCalendarCore/DecimalTime.swift

```diff
@@ -69,7 +69,15 @@ public struct DecimalTime {
 public extension DecimalTime {
     /// Initializes a new DecimalTime with the current time
     init(base: Date = Date()) {
-        let midnight = Calendar.gregorian.startOfDay(for: base)
+        self.init(base: base, timeZone: nil)
+    }
+
+    init(base: Date = Date(), timeZone: TimeZone?) {
```

Comment:
you could just put a default argument value of nil for timeZone instead of creating another initialiser